bugs.txt

```diff
@@ -327,6 +327,10 @@ bpo-44394:
     date: 2021-06-11 14:14:07
     title: '[security] CVE-2013-0340 "Billion Laughs" fixed in Expat >=2.4.0: Update
         vendored copy to expat 2.4.1'
+bpo-44549:
+    author: siddhartha shankar mahato
+    date: 2021-07-02 10:46:07
+    title: Update Windows installer to use bzip2 1.0.8
```

Comment:
Did you run make update? That's the outdated bpo number, you can remove it, it's no longer needed.

Reply:
> Did you run make update? That's the outdated bpo number, you can remove it, it's no longer needed.

I got an `SSLError` when executing `make update` on Ubuntu 22.04. I temporarily solved this problem by setting the `verfiy` parameter to False, although this is not a good solution.

```python
    def _get_gh_bug(self, number):
        print(f"Get GitHub issue #{number}")
        if self.github_api is None:
            self.github_api = github.Github(GITHUB_API_TOKEN)  # raise SSLError, set verfiy=False
        project = self.github_api.get_repo(GITHUB_REPO)
        issue = project.get_issue(number)
...
```

```
...

Traceback (most recent call last):
  File "/home/z00522458/workspace/python-security/render_doc.py", line 1248, in <module>
    main()
  File "/home/z00522458/workspace/python-security/render_doc.py", line 1238, in main
    app.main(yaml_filename, rst_filename)
  File "/home/z00522458/workspace/python-security/render_doc.py", line 1189, in main
    vulnerabilities = self.load_vulnerabilities(yaml_filename)
  File "/home/z00522458/workspace/python-security/render_doc.py", line 1103, in load_vulnerabilities
    vuln = Vulnerability(self, data)
  File "/home/z00522458/workspace/python-security/render_doc.py", line 617, in __init__
    raise Exception("failed to parse %r: %s" % (self.name, exc))
Exception: failed to parse 'Windows: vulnerable bzip2 1.0.6': HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/python/cpython (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')))
make: *** [Makefile:20: update] Error 1
```